types/fluxible-addons-react/fluxible-addons-react-tests.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom/server';
+import * as Fluxible from 'fluxible';
+import BaseStore = require('fluxible/addons/BaseStore');
+import { createElementWithContext } from './index';
+import connectToStores = require('./connectToStores');
+import { ComponentContext, ActionCreator } from '../fluxible';
+
+// Action
+const someAction: ActionCreator<{ say: string }> = async (ctx, payload) => {
+    await ctx.executeAction(subAction, {});
+    ctx.dispatch('ACTION', {});
+};
+
+const subAction: ActionCreator = async (ctx) => {
+    ctx.dispatch('SUB_ACTION', {});
+};
+
+// Component
+class App extends React.Component<{ label: string, amount: number }> {
+    context: ComponentContext;
+
+    componentDidMount() {
+        this.context.executeAction(someAction, { say: 'meow' });
+    }
+}
+
+// Store
+class TestStore extends BaseStore {
+    getText(): string { return 'meow'; }
+}
+
+const app = new Fluxible();
+const context = app.createContext();
+
+const ConnectedWithComponent = connectToStores(App, [TestStore], (context) => ({
```

Comment:
`ConnectedWithComponent: any` 🤔

Reply:
You mean the result is an implicit any? o_O

It is missing the generic parameter, though, and it can't infer from your function because you didn't tell TS what the first argument's type is. The tsconfig does have noImplicitAny enabled, though, so it should be a build error instead 🤔